docs/activities/test-resources/index.md

```diff
@@ -0,0 +1,34 @@
+# Resources for Testing
+
+## Test Environments
+There are a number of Test Instances available and the following table compares their features, the most accurate representation of the Production System are CIN4 and the Model office.
+![A table of test environments and their features](./CMS-environments.xlsx)
```

Comment:
Should this reference the image rather than the `xlsx` file?

Reply:
I got an update from Michael with the original spreadsheet which contains a fuller definition of the environments, so I've switched to the spreadsheet but forgotten to remove the '!'. Will make that change